app/connectors/connectors/jms/activemq-connectors/activemq-model/src/main/java/io/syndesis/connector/jms/TrustAllTrustManager.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.jms;
+
+import java.net.Socket;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+import javax.net.ssl.SSLEngine;
+import javax.net.ssl.X509ExtendedTrustManager;
+
+/**
+ * TrustManager that trusts all certificates.
+ * Use only for AMQ connections to SSL Brokers, when no Broker certificate provided.
+ * @author dhirajsb
+ */
+class TrustAllTrustManager extends X509ExtendedTrustManager {
```

Comment:
I would not do that, we're enabling MITM

Reply:
yes, you are right. But this is required for AMQ connections to Brokers using SSL. Without it, a certificate is always required. Which is not an easy thing to do. 
But, this was a requirement according to @chirino.

Reply:
This is a good option to have. All AMQ brokers on openshift can only be accessed via tls. So self signed certs at common in Dev envs

Reply:
This is a good option to have. All AMQ brokers on openshift can only be accessed via tls. So self signed certs at common in Dev envs